packages/vue/src/components/verify-user.vue

```diff
@@ -25,6 +27,25 @@ const {
   getVerifyContactText,
 } = authenticatorTextUtil;
 
+const censorContactInformation = (
```

Comment:
This is a copy/paste from the React library.

Reply:
Can we move the `censorContactInformation` util to the `ui` package and import from there instead of duplicating? Duplication increases potential for drift between the utils

Reply:
Also curious if this util lives in the Angular package?

Reply:
Yeah, that will make it easier to add tests. I'll take a look at Angular.